python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -12,10 +12,28 @@
 
 from .infra import LoadingGaugeType
 
-RAILJSON_ROLLING_STOCK_VERSION_TYPE = Literal["3.2"]
+RAILJSON_ROLLING_STOCK_VERSION_TYPE = Literal["3.3"]
 RAILJSON_ROLLING_STOCK_VERSION = get_args(RAILJSON_ROLLING_STOCK_VERSION_TYPE)[0]
 
 
+class RollingStockCategory(str, Enum):
+    """
+    This enum defines the rolling stock category that a train can belong to.
```

Comment:
```suggestion
    The list of categories that can be assigned to a rolling stock.
```

The term "train" can refer to too many things in OSRD's context, we should avoid using it on its own as much as possible imo. (In most contexts it actually refers to a train schedule instance.)

Reply:
Done.